include/wrp_ros/peripheral/lift_server_node.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file action_server.hpp
+ * @brief This file contains the implementation of a simple action and service
+ *        server for lift action. It communicates with an Arduino via a serial
+ *        port to transmit messages (e.g., goals, feedback request) and receive
+ *        feedback.
+ *
+ * @date 06-02-2024
+ *
+ * @copyright Copyright (c) 2024 Weston Robot Pte. Ltd.
+ */
+
+#ifndef LIFT_SERVER_NODE
+#define LIFT_SERVER_NODE
+
+#include <iostream>
+#include <fstream>
+
+#include <ros/ros.h>
+#include <wrp_ros/LiftQuery.h>
+#include <actionlib/server/simple_action_server.h>
+#include <wrp_ros/LiftControlAction.h>
+
+#include "wrp_sdk/peripheral/lift_controller.hpp"
+#include "wrp_sdk/interface/lift_interface.hpp"
+
+namespace westonrobot {
+class ActionServer {
+ public:
+  ActionServer();
+  ~ActionServer();
+
+  bool SrvCB(wrp_ros::LiftQuery::Request& req,
```

Comment:
Name should be more descriptive. 
```suggestion
  bool QueryCallback(wrp_ros::LiftQuery::Request& req,
```